app/serializers/book/show_serializer.rb

```diff
@@ -0,0 +1,8 @@
+class BookShowSerializer < BookSerializer
+  attribute :image, key: :image_url
+  attribute :actual_rent
```

Comment:
There is no need to do this in separated lines

Reply:
When you use a 'key', you need use attribute for this